subcommands/el2g/devices.go

```diff
@@ -26,9 +28,18 @@ func init() {
 fioctl el2g devices 
 
 # Show details of a device
-fioctl el2g devices  <device-id>
+fioctl el2g devices <device-id>
+
+# Add a device with an se050 (product ID: 935389312472)
+# Device ID can be found on a device by running:
+#  $ ssscli se05x uid | grep "Unique ID:" | cut -d: -f2
+#  ssscli se05x uid | grep "Unique ID:" | cut -d: -f2
+#  04005001eee3ba1ee96e60047e57da0f6880
+# This ID is hexadecimal and must be prefixed in the CLI with 0x. For example:
+fioctl el2g devices --add 935389312472 0x04005001eee3ba1ee96e60047e57da0f6880
 `,
 	}
+	devicesCmd.Flags().StringVarP(&add, "add", "", "", "Whitelist the device for the given nc12 product id")
```

Comment:
I don't understand from the help message what this command really does.

It also looks strange that it is implemented as a flag `--add` and not as a subcommand `add`.

Reply:
i've grown sick of the `cmd list|add|show|del`. but i guess i should re-work that to make it consistent.

Reply:
I think the `cmd list|add|show|del` is that case when it is 10% harder to write the initial code, but 90% easier to support it later.

Reply:
ah, that...

I think one day we could hack the `cobra` to default `fioctl devices` to `fioctl devices list`, and make that change for all commands where it is relevant.